aws-bootstrap/create_service_account.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+set -e  # Exit on error
+
+SERVICE_ACCOUNT_NAME="my-service-account"
+POLICY_ARN="arn:aws:iam::aws:policy/AmazonQDeveloperAccess"
```

Comment:
@prattsvee Again, open the pull request instead of telling me it's ready. Now I've opened it to submit your work, which makes no sense, as I'm reviewing myself here -.-

Anyway, 

Very well done. Only comment is to iterate over a list of POLICY_ARNS and attach them in a similar manner that you are iterating and removing them.

The current page and attached roles for the **terraform** user. Replicate the ability to create a user and attach any number of ARNS via a Group. 

We'll handle that in another issue, of which I presume you will open a PR lol.

Reply:
Sorry I thought I raised a PR , Will work on raised a PR also next time , Lol.